02 - Community Expansions/02.01 Plugins by Category/Refactoring and auto-formatting plugins.md

```diff
@@ -0,0 +1,35 @@
+---
+aliases:
+- 
+tags: 
+- seedling 
+publish: true
+---
+
+
+# Refactor
+
+Plugins that offer additional functionality to reformat notes than the [[Obsidian Core Plugins#Note composer]] plugin. These plugins usually make automated changes to your notes.
```

Comment:
I think there is a word missing in this sentence 🤔

Reply:
Rephrased the description for this note